### PR TITLE
latitude and longitude on devices/device fetch

### PR DESCRIPTION
**Pull request by sorribes22**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: html/includes/api_functions.inc.php [hunk 1]
```diff
@@ -347,7 +347,7 @@ function list_devices()
         $param[] = $_SESSION['user_id'];
     }
     $devices = array();
-    $dev_query = "SELECT $select FROM `devices` AS d $join WHERE $sql GROUP BY d.`hostname` ORDER BY $order";
+    $dev_query = "SELECT dev.*, lat, lng FROM (SELECT $select FROM `devices` AS d $join WHERE $sql GROUP BY d.`hostname` ORDER BY $order) dev LEFT JOIN locations ON locations.location=dev.location";
```
Comment (laf) [inline html/includes/api_functions.inc.php]: Revert this line back to how it was and use this patch instead:

https://p.libren.ms/view/raw/d23ae0fb

**Comment by sorribes22** [comment]
A lot of thanks for your help @laf. Is merging blocked until I sign as "axsor"? Because I will need to change github username now. If is necessary I will do it temporary.

**Comment by laf** [comment]
Yes it is but if you squash your commit to use the first name then it will be fine.

**Comment by sorribes22** [comment]
I close and reopen de pull request because I can't resolve it...

**Comment by sorribes22** [comment]
Really...

**Pull request closed without merging**